pkg/apis/migration/v1alpha1/resource.go

```diff
@@ -70,6 +71,26 @@ func (r *MigPlan) Touch() {
 	r.Annotations[TouchAnnotation] = uuid.New().String()
 }
 
+func (r *MigPlan) GetVerify() (verify bool) {
```

Comment:
The annotation is no longer needed.  Just a `bool`.

Reply:
Sorry, this part was reintroduced after rebase, will clean it up.